src/utility/sdk.js

```diff
@@ -105,7 +105,10 @@ export const initSdk = async () => {
       const event = new CustomEvent("snetMMNetworkChanged", { detail: { network } });
       window.dispatchEvent(event);
     });
-    updateSDK();
+    window.web.eth.accounts().then(accounts => {
```

Comment:
It should be `window.web3.eth`

Reply:
https://github.com/singnet/refactored-dapp/pull/280/commits/fc6c4ab98efee3a85be45eb8b60077e706af93ed